src/components/VxpPanel.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <StackLayout>
+    <StackLayout class="vxp-panel-title">
+      <VxpLabel v-if="title" class="vxp-panel-title__title" :text="title"></VxpLabel>
+      <slot name="topbar"></slot>
+    </StackLayout>
+    <StackLayout class="vxp-panel-content">
+      <slot name="content"></slot>
+    </StackLayout>
+  </StackLayout>
+</template>
+
+<script>
+import StackLayout from '../layouts/StackLayout';
+import VxpLabel from './VxpLabel';
+
+export default {
+  name: 'VxpPanel',
+  components: {
+    StackLayout,
+    VxpLabel,
+  },
+  props: {
+    title: String,
+  },
+  methods: {},
+};
+</script>
+
+<style lang="scss" scoped>
+@import './../assets/styles/helpers.scss';
+.vxp-panel-content {
+  background-color: #f4f5fa;
+  padding: unit(0) unit(15);
+}
+.vxp-panel-title {
+  display: flex;
+  height: 40px;
```

Comment:
unit function should be used instead of px

Reply:
Changed with unit function